core-services/08-sysctl.sh

```diff
@@ -2,15 +2,8 @@
 
 if [ -x /sbin/sysctl -o -x /bin/sysctl ]; then
     msg "Loading sysctl(8) settings..."
-    for i in /run/sysctl.d/*.conf \
-        /etc/sysctl.d/*.conf \
-        /usr/local/lib/sysctl.d/*.conf \
-        /usr/lib/sysctl.d/*.conf \
-        /etc/sysctl.conf; do
-
-        if [ -e "$i" ]; then
-            printf '* Applying %s ...\n' "$i"
-            sysctl -p "$i"
-        fi
-    done
+    system-dirs sysctl.d | tr '\012' '\0' | xargs -0 -I{} sysctl -p {}
```

Comment:
This is a boot time dependency on tr and xargs, which while it should be available the previous version didn't have.  Can you get around this?

Reply:
I can use a for loop, but I am not sure if there are any speed considerations. Considering modules loading (which is done even earlier, before filesystems are mounted) uses them, I am not sure how critical it is to avoid.

Reply:
used a for loop